.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto eol=lf
```

Comment:
Why not auto per [GitHub's recommendations](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings)?

Reply:
Not everyone apply this `git config --global core.autocrlf true`.
This file is applied when commiting.